### PR TITLE
Fixes for rFactor

### PR DESCRIPTION
**Pull request by dallongo**
Fixed tire temp warnings; Fixed pit exit traffic notifications; Added black flag notification; Adjusted blue flag behavior; Adjusted invalid lap detection; Added ambient temps, track temps, and wind info; Added detached wheel info; Fixed auto-launch; Added separate menu items for Automobilista and Stock Car Extreme; Adjusted auto-launch options for R3E.

### CODE REVIEW

File: CrewChiefV4/App.config [hunk 1]
```diff
@@ -100,14 +100,11 @@
...
-      <setting name="r3e_launch_params" serializeAs="String">
+      <setting name="r3e_steam_id" serializeAs="String">
         <value>-applaunch 211500</value>
       </setting>
-      <setting name="r3e_launch_exe" serializeAs="String">
-        <value>c:/games/steam/steam.exe</value>
-      </setting>
       <setting name="launch_raceroom" serializeAs="String">
-        <value>False</value>
+        <value>True</value>
```
Comment (mrbelowski) [inline CrewChiefV4/App.config]: This default value should be 'false'

File: CrewChiefV4/App.config [hunk 2]
```diff
@@ -356,14 +353,26 @@
...
+        <value>-applaunch 339790</value>
       </setting>
-      <setting name="rf1_launch_exe" serializeAs="String">
-        <value />
+      <setting name="steam_exe" serializeAs="String">
+        <value>C:/Program Files (x86)/Steam/Steam.exe</value>
       </setting>
       <setting name="launch_rfactor1" serializeAs="String">
-        <value>False</value>
+        <value>True</value>
```
Comment (mrbelowski) [inline CrewChiefV4/App.config]: not sure if you want this to be 'true'

File: CrewChiefV4/App.config [hunk 3]
```diff
@@ -100,14 +100,11 @@
       <setting name="custom_device_guid" serializeAs="String">
         <value />
       </setting>
-      <setting name="r3e_launch_params" serializeAs="String">
```
Comment (mrbelowski) [inline CrewChiefV4/App.config]: i think I understand what you've done here, but I think we need to be able to specify launch params as well as the steam ID - some users need the -cefNoGpu or -noReplay params, as well as the steam app ID

File: CrewChiefV4/App.config (hunk 2 above)
Comment (dallongo) [inline CrewChiefV4/App.config]: In the forums it appears that the average user is expecting the default behavior to launch the game automatically without further configuration. I agree though that as a rule, 'False' is a much safer default. I'll change it back to 'False' and make a note in the installation procedures on the forums.

File: CrewChiefV4/App.config (hunk 1 above)
Comment (dallongo) [inline CrewChiefV4/App.config]: This must have gotten toggled by accident when I was playing with the other settings. I'll change it back to 'False'.

File: CrewChiefV4/App.config (hunk 3 above)
Comment (mrbelowski) [inline CrewChiefV4/App.config]: my suggestion here is to rename these to '..._launch_params' for now, and reinstate the old help text (and anything else that was changed). We could do a separate param for steam app ID if we need to later on - keep this pull request's scope to AMS / rF1 fixes only if we can

Comment (dallongo) [inline CrewChiefV4/App.config]: As a rule, commandline options for Steam games should be specified within Steam by right-clicking on the title, going to properties, and setting the launch options there. Running via the Steam AppID should be the preferred method for Steam titles anyway.

Consolidating the settings helps reduce the clutter in the Properties page, especially with so many new users coming onboard for rFactor 1 support, I'm just trying to stem the tide of forum posts for support on simple stuff.

I agree that this PR should stick to just rF1 fixes though, and I'll change the setting names back to be consistent with R3E. Doing a piece-meal consolidation of the Properties page is likely to cause more harm than good at this stage anyway, and it's simple enough to do later.

**Pull request merged**